data_management/models.py

```diff
@@ -484,30 +479,19 @@ class StorageRoot(BaseModel):
     `updated_by`: Reference to the user that updated this record
     """
     EXTRA_DISPLAY_FIELDS = ('locations',)
-    ADMIN_LIST_FIELDS = ('name',)
 
-    PUBLIC = 0
-    PRIVATE = 1
-    CHOICES = (
-        (PUBLIC, 'Public'),
-        (PRIVATE, 'Private'),
-    )
-    name = NameField(null=False, blank=False)
     root = URIField(null=False, blank=False, unique=True)
-    accessibility = models.SmallIntegerField(choices=CHOICES, default=PUBLIC)
+    local = models.BooleanField(default=False)
 
     class Meta:
         constraints = [
             models.UniqueConstraint(
-                fields=('name',),
+                fields=('root', 'local'),
```

Comment:
Do we need to have a uniqueness constraint on `local` as well as root? I think we absolutely need to know whether it's local or not, but we may not need to allow the same root being local and remote?

Reply:
Ok, maybe we should remove `local` from the uniqueness constraint for now. If a use case comes up later where someone does need the same root to be both local and remote we can reconsider this...